frontend/App.js

```diff
@@ -56,6 +56,32 @@ const App = () => {
     "PlusJakartaSans-Bold": require("./assets/fonts/PlusJakartaSans-Bold.ttf"),
     "PlusJakartaSans-ExtraBold": require("./assets/fonts/PlusJakartaSans-ExtraBold.ttf"),
   });
+  const [initialRoute, setInitialRoute] = React.useState('Login');
+
+  React.useEffect(() => {
+    checkTokenValidity();
+  }, []);
+
+  const checkTokenValidity = async () => {
```

Comment:
I am still seeing login option if i am refreshing the page

Reply:
ok wait i will check and update.